flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/adaptive/ExecutingTest.java

```diff
@@ -594,8 +683,12 @@ public FailureResult howToHandleFailure(Throwable failure) {
         }
 
         @Override
-        public boolean shouldRescale(ExecutionGraph executionGraph) {
-            return canScaleUp.get();
+        public boolean shouldRescale(ExecutionGraph executionGraph, boolean forceRescale) {
+            if (forceRescale) {
+                return canScaleUpWithForce;
+            } else {
+                return canScaleUpWithoutForce;
```

Comment:
From `@Nullable private Boolean canScaleUpWithForce = false;`, we can see the `canScaleUpWithForce` can be null.

It will be `NullPointException` once it's null here, because the return value is `boolean` instead of `Boolean`.

I prefer remove the `Boolean`, and still mark the default value is false. WDYT?

Reply:
I was trying to stick to nullable as in the production code and also because in some tests we don't care about forcing value. But yes I can definitely move to boolean instead of Boolean to avoid NPE when unboxing.

Reply:
done